### PR TITLE
DOCS-1844: Adds doc for security event webhooks

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-enterprise/threat/configuring-webhooks.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+description: Get security event alerts in Slack or Jira.
+title: Configuring Security Event alerts in Slack and Jira
```
Comment (gantony) [inline calico-enterprise/threat/configuring-webhooks.mdx]: nit: there is some inconsistencies aroud nthe capitulisation of Security Events. Not sure how best to resolve it.
image

File: calico-enterprise/threat/configuring-webhooks.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## Before you begin
+
+Your target application must be configured to receive data from the {{prodname}} webhook.
+
+* **Slack**. You must have a webhook URL for the Slack app that you want {{prodname}} to send alerts to.
+See [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/webhooks) for more information.
+* **Jira**. You must have an API token for an Atlassian user account that has write permissions to your Jira instance.
+  See [Manage API tokens for your Atlassian account](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/) for details on how to obtain an API token.
+  You also need:
+  * Your Atlassian site URL. If you access Jira at the URL `https://<your-company>.atlassian.net/jira`, then your site URL is `<your-company>.atlassian.net`.
```
Comment (gantony) [inline calico-enterprise/threat/configuring-webhooks.mdx:26]: @ctauchen, could you please change `<your-company>.atlassian.net` to be `https://<your-company>.atlassian.net`. I feel like this fits better with what people expect of a URL and is a bit more consistent with the rest of the UI. Planning to including `https://` for the field in the UI.

**Review by gantony — APPROVED** [review]
Thanks @ctauchen, LGTM! I've added 2 small comments please :pray:

File: calico-enterprise/threat/configuring-webhooks.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-enterprise/threat/configuring-webhooks.mdx]: Thanks, will take a look.

File: calico-enterprise/threat/configuring-webhooks.mdx (hunk 2 above)
Comment (ctauchen) [reply calico-enterprise/threat/configuring-webhooks.mdx:26]: Main thing is that this is consistent with what the UI needs. I think without the protocol is better, but we can discuss.

**Pull request merged**